internal/kuberesource/sets.go

```diff
@@ -627,3 +627,159 @@ sleep inf
 
 	return []any{vss}
 }
+
+// MySQL returns the resources for deploying a MySQL database
+// with an encrypted luks volume using the workload-secret.
+func MySQL() []any {
+	initCommand := `#!/bin/bash
+# cryptsetup complains if there is no /run directory.
+mkdir /run
```

Comment:
Consider adding an `emptyDir` mount instead.

Reply:
We also already have the same(?) script in https://github.com/edgelesssys/contrast/blob/main/internal/kuberesource/sets.go#L504. We likely want to build a function returning the string (maybe add some templating if needed) and use it in both places.